odl/operator/oputils.py

```diff
@@ -319,6 +319,76 @@ def rmatvec(v):
                                               dtype=dtype)
 
 
+def as_scipy_functional(func, return_gradient=False):
+    """Wrap ``op`` as a function operating on linear arrays.
+
+    This is intended to be used with the scipy solvers.
+
+    Parameters
+    ----------
+    func : `Functional`
+        A functional that should be wrapped
+    return_gradient : bool, optional
+        If the gradient of the functional should also be returned
```

Comment:
dot

We abandoned this style, didn't we? In favor of "\`\`True\`\` if \<condition fulfilled\>, \`\`False\`\` otherwise."